### ISSUE TITLE
Key rotation

### ISSUE THREAD

**Issue by i5ar** [qmk/qmk_configurator#310]
The QMK Firmware repository have a keyboard defining rotations (https://github.com/qmk/qmk_firmware/blob/master/keyboards/signum/3_0/elitec/info.json) but they don't seem to be correct (https://i5ar.github.io/yake/#/signum/3_0/elitec). I implemented rotations (`r`, `rx` and `ry`) the same way KLE does (which is the same way SVG does and probably other libraries too).
I can make a PR to fix the rotations of that keyboard but before I want to make sure the QMK Configurator will also be implementing `rx` and `ry` the same way.

This is the [fix](https://github.com/i5ar/qmk_firmware/blob/fix/keyboards/signum/3_0/elitec/info.json). You can load it [here](https://i5ar.github.io/yake) to see the changes.

**Comment by yanfali** [comment]
Yeah go ahead if it's not correct compared to others. We're not using rotation right now.
